extension/manifest.json

```diff
@@ -25,6 +25,17 @@
         "48": "images/GearHeart_Mixed_47x47.png",
         "128": "images/GearHeart_Mixed_127x127.png"
     },
+    // added a keyboard shortcut or command to open the panel
```

Comment:
You can't have comments in JSON files.

Reply:
Sorry, I mistakenly included comments in JSON file.